functions/search.js

```diff
@@ -0,0 +1,64 @@
+import knex from 'knex';
+
+const db = knex({
+  client: 'pg',
+  connection: {
+    host: 'opensgid.agrc.utah.gov',
+    user: 'agrc',
+    password: 'agrc',
+    database: 'opensgid',
+    port: 5432,
+  },
```

Comment:
**polish**: this might get some attention from bad actors. What do you think about tossing it in a secret?

Reply:
Ya, I wondered about that. I'll do it.